pystiche_papers/li_wand_2016/_data.py

```diff
@@ -18,11 +18,6 @@ def make_image_transform(image: str) -> transforms.ComposedTransform:
     if image == "emma":
         image_transform = transforms.Crop(origin=(30, 12), size=(930, 682))
     elif image == "jenny":
-
-        class MirrorHorizontally(transforms.Transform):
```

Comment:
Same as above. While this is not as specific, I don't think another paper will use it.